scanomatic/generics/abstract_model_factory.py

```diff
@@ -711,10 +711,10 @@ def get_next_free_section(
         while my_section in sections:
             my_section = section.format(
                 section_name,
-                " #{0}".format(enumerator) if enumerator else enumerator
+                " #{0}".format(enumerator) if enumerator is not None else ''
             )
             if my_section in sections:
-                if enumerator:
+                if enumerator is not None:
```

Comment:
is there a place where enumerator can be 0?

Reply:
No, as I read it it always starts on 2 and goes up. I guess you suggest just initialising it to 0?

Reply:
no it was just that `if enumerator` would be false when enumerator is `0`, so you would have changed the logic but if you are sure it can't be 0 then 👍🏾